pygmt/src/timestamp.py

```diff
@@ -23,27 +23,38 @@ def timestamp(
     r"""
     Plot the GMT timestamp logo.
 
+    Add the GMT timestamp logo with an optional label at the bottom-left corner
+    of a plot. The timestamp will be in the locale set by the environment
+    variable **TZ** (generally local time) and its format is controlled by the
```

Comment:
```suggestion
    variable **TZ** via ``os.environ["TZ"]`` (generally local time) and its format is controlled by the
```
As is it's not clear how to change that variable (via os).

Reply:
I'm unsure about this change, because it's also possible to set TZ using `export TZ=xxx` or other ways.

Reply:
I see, however, I feel it's not fully clear without an expanded description.

Reply:
OK, I've changed it to 

> The timestamp will be in the locale set by the environment variable **TZ** (generally local time but can be changed via ``os.environ["TZ"]``)